src/main/java/games/strategy/triplea/settings/ClientSettingSwingUiBinding.java

```diff
@@ -138,8 +138,9 @@
   TEST_LOBBY_PORT_BINDING(
       "Lobby Port Override",
       SettingType.TESTING,
-      SelectionComponentFactory.intValueRange(ClientSetting.TEST_LOBBY_PORT, 1, 99999),
-      "Specifies the port for connecting to a test lobby."),
+      SelectionComponentFactory.intValueRange(ClientSetting.TEST_LOBBY_PORT, 1, 99999, true),
```

Comment:
Just noting for another PR that the upper bound on the port should probably be 65535 instead of 99999.

Reply:
This is a minor change...
Will do, good suggestion